apps/system-e2e/src/fixtures/applications/users.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "Gervimaður Afríka",
+    "mobile": "0103019",
```

Comment:
```suggestion
    "phoneNumber": "0103019",
```

Reply:
Not seeing it on main, so will delay that until it lands there I suppose.